test/test_pty.rs

```diff
@@ -0,0 +1,36 @@
+use nix::pty::openpty;
+use nix::unistd::{read, write, close};
+
+#[test]
+fn test_openpty() {
+    // TODO: figure out the right termios settings to pass in here
```

Comment:
Yeah, `tcgetattr(stdin)` is failing on OS X, added two print!() statements around that call and only the first one shows up in the log. Two obvious options are to either disable the test on OS X or switch back to just using `(None, None)`